gspread/worksheet.py

```diff
@@ -3201,3 +3204,88 @@ def cut_range(
         }
 
         return self.client.batch_update(self.spreadsheet_id, body)
+
+    def add_validation(
+        self,
+        source: str,
+        condition_type: ValidationConditionType,
+        values: Iterable[Any],
+        *,
```

Comment:
I don't understand why is this unnamed argument here then if we have an actual list of values into a single argument.

Reply:
I checked-out your branch, and the tests are failing because of the `*` argument.

you need to remove it.
currently the arguments:
- self
- ~source~ range
- condition_type
- values

are regular args with no default value.

The arguments:
- inputMessage
- strict
- showCustomUi

are kwargs, because they have a default value set.